.github/workflows/test.yml

```diff
@@ -444,6 +452,42 @@ jobs:
         shell: bash
         run: corepack disable
 
+  docs:
+    name: Docs Checks
+    timeout-minutes: 30
+    if: needs.determine_jobs.outputs.docs == 'true'
+    needs: [determine_jobs]
+    runs-on: ubuntu-latest
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+      TURBO_REMOTE_ONLY: true
+
+    steps:
+      # on main -> current + prev commit
+      # pr -> pr commits + base commit
+      - name: Determine fetch depth
+        id: fetch-depth
+        run: |
+          echo "depth=$(( ${{ github.event.pull_request.commits || 1 }} + 1 ))" >> $GITHUB_OUTPUT
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: ${{ steps.fetch-depth.outputs.depth  }}
+
+      - name: Build turborepo CLI from source
```

Comment:
This name isn’t accurate anymore, it doesn’t build turborepo. Also you don’t need all the things this custom action does, you just need the global turbo install